openghg/standardise/footprint/_paris.py

```diff
@@ -0,0 +1,175 @@
+import logging
+from pathlib import Path
+from collections import defaultdict
+from typing import DefaultDict, Dict, List, Optional, Union
+import warnings
+from xarray import Dataset
+
+from openghg.util import (
+    check_species_time_resolved,
+    check_species_lifetime,
+    timestamp_now,
+    check_function_open_nc,
+)
+from openghg.store import infer_date_range, update_zero_dim
+from openghg.types import multiPathType
+
+logger = logging.getLogger("openghg.standardise.footprint")
+logger.setLevel(logging.DEBUG)  # Have to set level for logger as well as handler
+
+
```

Comment:
Here or in the init file, you could add `parse_flexpart = parse_paris` and `parse_name = parse_paris`. I think that is all that is needed to "alias" this function.

Reply:
I'm not sure about aliasing this for name, in particular, as this is still a specific format from NAME that isn't necessarily universal yet. For our NAME footprints in particular we still have previous format files so either "acrg_org" or "paris" may be valid depending on the file itself. We could perhaps add some fail cases to advise which format to use to help (e.g. check for "srr" or "fp" and mention using the other format).

For flexpart to keep this consistent with the original PR I can add that but, again, in future we may want to think about whether that's universal.